pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/assets/manifests/triliovault-ovh-s3-target.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Target
+metadata:
+  name: trilio-ovh-s3-target
+spec:
+  type: ObjectStore
+  vendor: Other
+  objectStoreCredentials:
+    url: "https://<YOUR_OVH_OBJECT_STORAGE_BUCKET_ENDPOINT_URL_HERE>"
+    bucketName: "<YOUR_OVH_STORAGE_BUCKET_NAME_HERE>"
+    region: "<YOUR_OVH_OBJECT_STORAGE_BUCKET_REGION_HERE>"
+    credentialSecret:
+      name: trilio-ovh-s3-target-secret
```

Comment:
change 
`trilio-ovh-s3-target-secret`
to
`trilio-ovh-s3-target`

Reply:
The `credentialSecret.name` is correct as `trilio-ovh-s3-target-secret`
`trilio-ovh-s3-target` is the name of the target `metadata.name`

Reply:
So you must change the Secret definition in the main page:

To access S3 storage, each target needs to know bucket credentials. A Kubernetes Secret must be created as well:

```
apiVersion: v1
kind: Secret
metadata:
  name: trilio-ovh-s3-target
  namespace: tvk
type: Opaque
stringData:
  accessKey: <YOUR_OVH_OBJECT_STORAGE_BUCKET_ACCESS_KEY_ID_HERE>	# value must be base64 encoded
  secretKey: <YOUR_OVH_OBJECT_STORAGE_BUCKET_SECRET_KEY_HERE>    	# value must be base64 encoded
```

Reply:
Thank you @raclepoulpe for pointing this out. I have corrected the `metadata.name` at all places in `guide.en-us.md` file